packages/internal/src/ts2js.ts

```diff
@@ -57,8 +57,11 @@ export const convertTsFilesToJs = (cwd: string, files: string[]) => {
  */
 export function convertTsConfigsToJsConfigs(cwd = getPaths().base) {
   // Should find 'api/tsconfig.json`, `web/tsconfig.json`, and `scripts/tsconfig.json`.
-  const tsConfigs = fg.sync('*/tsconfig.json', { cwd, absolute: true, ignore: '**/node_modules'
- })
+  const tsConfigs = fg.sync('*/tsconfig.json', {
+    cwd,
+    absolute: true,
+    ignore: ['node_modules'],
```

Comment:
I'm assuming you've checked this, but maybe it should be `**/node_modules`?

Reply:
Yeah when I run it only I find the three tsconfigs we expect. I'm not using the globstar, only the single start (not sure what it's called—`*`), so I'm only matching one directory, one-level deep. So even if we didn't ignore `node_modules`, we wouldn't be searching through every package inside `node_modules`. I.e. to get a match in node_modules if we didn't ignore it, there'd have to be a package called `tsconfig.json`.